src/playercards/cards/FavoroftheSun1.ttslua

```diff
@@ -6,3 +6,26 @@ SHOW_SINGLE_RELEASE = true
 KEEP_OPEN = true
 
 require("playercards/CardsThatSealTokens")
+
+local playmatApi = require("playermat/PlaymatApi")
+local guidReferenceApi = require("core/GUIDReferenceApi")
+local chaosBagApi      = require("chaosbag/ChaosBagApi")
+local contextMenuAdded
+
+
+function onCollisionEnter()
+  if contextMenuAdded then return end
+  contextMenuAdded = true
+  self.addContextMenuItem("Resolve sealed Bless", resolveSealed)
+end
+
+function resolveSealed()
+    if #sealedTokens == 0 then
+      broadcastToAll("No tokens sealed.", "Red")
+      return
+    end
+    local closestMatColor = playmatApi.getMatColorByPosition(self.getPosition())
+    local mat = guidReferenceApi.getObjectByOwnerAndType(closestMatColor, "Playermat")
+    guidToBeResolved = table.remove(sealedTokens)
+    chaosBagApi.drawChaosToken(mat, true, _, guidToBeResolved)
+end
```

Comment:
Since this code is identical for the Favors and the Codex, it would be better if it is put somewhere else (either in a separate file or part of the library file - in the latter case this would be activated by some variable in this script like the other things)